cmake/modules/FindSaxon.cmake

```diff
@@ -50,11 +50,11 @@ if (JAVA_RUNTIME)
   # Find Saxon 6.5.x
   #
   find_file (SAXON
-    NAMES saxon.jar saxon6.jar saxon-6.5.5.jar saxon-6.5.4.jar saxon-6.5.3.jar
+    NAMES saxon.jar saxon6.jar saxon-6.5.5.jar saxon-6.5.4.jar saxon-6.5.3.jar saxon9he.jar
```

Comment:
Did you try if saxon9 works? At least for the usermanual Ulrich mentioned that only version 6 is usable.